### DISCUSSION TITLE
Centos 7 log file

### DISCUSSION THREAD

**Discussion by swilsonnc** [GerryDazoo/Slinger#447] [General]
If you run the service on Centos 7 using the default sling.service file and your systemd version is not 236 then it will not log the output.  To fix this edit the sling.service file and change these lines.

//StandardOutput=append:/tmp/sling.log
//StandardError=append:/tmp/sling.log
ExecStart=/bin/sh -c 'exec /usr/bin/env python -u slingbox_server.py >> /tmp/sling.log 2>&1'

Your paths may be different.

You must run systemctl daemon-reload after changing the file.

**Comment by GerryDazoo** [comment]
You are correct sir. The one supplied by me was for the RaspberryPi.
